app/views/admin/index.html.haml

```diff
@@ -14,21 +14,23 @@
   %h2 Useful links
   %ul
     %li
-      %a{href: '/article_finder'} Article Finder
+      %a{href: '/faq/new'} Add new FAQ
     %li
-      %a{href: '/usage'} Usage
+      %a{href: '/article_finder'} Article Finder
     %li
-      %a{href: '/sidekiq'} Job queue
+      %a{href: '/copy_course'} Copy Course from Another Dashboard Server
```

Comment:
'Copy course from another server'. No need for extra capitalization, and I think 'Dashboard' is redundant.

Reply:
Done on 048de6856118b649e1f5d97e595e75c18fdaf3fa